crates/runtime/src/capability/builtin.rs

```diff
@@ -249,6 +262,26 @@ pub enum TargetInterface {
     },
 }
 
+impl TargetInterface {
+    /// Returns the 3-tuple of (namespace, package, interface) for this interface
+    #[must_use]
+    pub fn as_parts(&self) -> (&str, &str, &str) {
+        match self {
+            Self::WasiBlobstoreBlobstore => ("wasi", "blobstore", "blobstore"),
+            Self::WasiHttpOutgoingHandler => ("wasi", "http", "outgoing-handler"),
+            Self::WasiKeyvalueAtomic => ("wasi", "keyvalue", "atomic"),
+            Self::WasiKeyvalueEventual => ("wasi", "keyvalue", "eventual"),
+            Self::WasiLoggingLogging => ("wasi", "logging", "logging"),
+            Self::WasmcloudMessagingConsumer => ("wasmcloud", "messaging", "consumer"),
```

Comment:
We're going to be able to eventually remove these right -- I figure avoiding breaking more things in this PR is the goal here (I did the same) for a `from_parts` IIRC